docs/user/install-gke-external.md

```diff
@@ -0,0 +1,229 @@
+# Turbinia External Access and Authentication Instructions
+
+## Introduction
+
+In this guide you will learn how to externally expose the Turbinia application.
+This guide is recommended for users who have already deployed Turbinia to a GKE
+cluster, but would like to access the Web UI and API server through an externally
+available URL instead of port forwarding the Turbinia service locally from the cluster.
+
+### Prerequisites
+
+- A Google Cloud Account and a GKE cluster with Turbinia deployed
+- The ability to create GCP resources
+- `gcloud` and `kubectl` locally installed
+
+## Deployment
+
+Please follow the steps below for configuring Turbinia to be externally accessible.
+
+### 1. Create a static external IP address
+
+- Create a global static IP address as follows:
+
+```
+gcloud compute addresses create turbinia-webapps --global
+```
+
+- You should see the new IP address listed:
+
+```
+gcloud compute addresses list
+```
+
+### 2. Create a domain or use pre-existing one
+
+You will need a domain to host Turbinia on. You can either register a new domain in a registrar
+of your choice or use a pre-existing one.
```

Comment:
I would maybe just title this section "Set up domain and DNS", and then have three options here that people can skip to for either 1) Registration via GCP, 2) Registration via Google Domains or 3) External Registrar and DNS provider.  Right now these three options are kind of spread out in this section and it might be easier to follow if they were separated out.

Reply:
Ack broke these sections out to make it more clear